Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.6.3
+
+ARG APP_DIR=/usr/src/app/
+
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+  apt-get update && \
+  apt-get -qq install -y \
+  build-essential \
+  libpq-dev \
+  nodejs
+
+RUN mkdir -p $APP_DIR
+
+WORKDIR $APP_DIR
+
+COPY Gemfile* $APP_DIR
+
+RUN bundle install
+
```

Comment:
```suggestion
COPY package.json $APP_PATH/
COPY yarn.lock $APP_PATH/
RUN yarn install --production --frozen-lockfile \
 && yarn install --check-files
```

Reply:
`frozen-lockfile` raises the following error
```
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
```

I figuring out how to build the project using alpine images though